docs/interpreter/tachyon.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: page
+title: "Tachyon Interpreter"
+description: "Tachyon Interpreter"
+group: manual
+---
+{% include JB/setup %}
+
+## Tachyon Interpreter for Apache Zeppelin
+[Tachyon](http://tachyon-project.org/) is a memory-centric distributed storage system enabling reliable data sharing at memory-speed across cluster frameworks.
+
+## Configuration
+<table class="table-configuration">
+  <tr>
+    <th>Name</th>
+    <th>Class</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>tachyon.master.hostname</td>
+    <td>localhostr</td>
```

Comment:
Hi @maocorte  : ) Thank you for the new interpreter in Zeppelin.
There is a small typo in here. Maybe **localhost** is right. isn't it?